packages/shared/src/modules/users/libs/types/user-sign-up-request-dto.type.ts

```diff
@@ -1,4 +1,5 @@
 type UserSignUpRequestDto = {
+	confirmPassword?: string;
```

Comment:
We shouldn't send confirmPassword to the backend

Reply:
so, how can i pass prop "name" from input component ?

![image](https://github.com/user-attachments/assets/ac1e670f-66bb-4594-8da6-1ce382a021f3)

Reply:
No need to have `confirmPassword` field for now